tests/cmux.py

```
@@ -752,6 +752,24 @@ def clear_ports(self, tab: str = None) -> None:
         if not response.startswith("OK"):
             raise cmuxError(response)
 
+    def set_agent_pid(self, key: str, pid: int, tab: str = None) -> None:
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)